spec/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'user'
+
+describe User do
```

Comment:
Make sure you are also testing the class methods interact with the database as expected.